Source/Common/GLTFUnarchiver.swift

```diff
@@ -771,7 +775,8 @@ public class GLTFUnarchiver {
         return valueArray
     }
     
-    private func loadImage(index: Int) throws -> Image {
+    // Returns either an Image or a URL
+    private func loadImage(index: Int) throws -> Any {
```

Comment:
Similar to [the above comment](https://github.com/magicien/GLTFSceneKit/pull/34/files#diff-15843db1746890d478f30fbca4e35f5cR39) regarding `Any`, I am willing to revise this part of the code to avoid `Any` as the return type. I would consider factoring out the shared logic here in `loadImage()` and then create a second `private func loadImageUrl(index: Int) throws => URL` method.